SUSE/scf-minibroker
# Moved minibroker tooling into this repository.

Modified to handle the changed directory hierarchy.

## Description

https://jira.suse.com/browse/CAP-566

Script `assess-minibroker-charts.rb` to perform assessment based on current state of master index of all stable helm chart.

Patch files required by charts to work with CAP/minibroker.

Documentation (Requisites, usage).

## Test plan

  - Create CAP cluster, point `kubectl` at it.
  - Run script as per its documentation. 
  - Wait two days (for completion), or just long enough to see a single working chart.
  - Generate yaml from main results, using the secondary script, as per its documentation.
  - :warning: The referenced ticket has a `results.yml` attached, my results. This enables testing in/of incremental mode as well, which is much faster. Just removing specific entries in the file can be used to trigger testing of known (non-)working charts.

:warning: At this point the tooling is not integrated into any type of pipeline.

See https://jira.suse.com/browse/CAP-600 for the relevant ticket and work.
